node/src/Commands.ts

```diff
@@ -884,3 +884,40 @@ export function createEcho(message: string): redis_request.Command {
 export function createPttl(key: string): redis_request.Command {
     return createCommand(RequestType.PTTL, [key]);
 }
+
+/**
+ * @internal
+ */
+export function createZremRangeByScore(
+    key: string,
+    minScore: ScoreLimit,
+    maxScore: ScoreLimit
+): redis_request.Command {
+    const args = [key];
+
+    if (minScore == "positiveInfinity") {
```

Comment:
This code is a duplication of createZcount. Lets move it to a separate function (something like getScoreLimitArg)